src/services/sig.py

```
@@ -168,16 +149,15 @@ def get_by_id(self, id: int) -> SIG:
             raise HTTPException(404, detail="해당 id의 시그/피그가 없습니다")
         return sig
 
-    def get_sig_response_by_id(self, id: int) -> SigResponse:
-        sig = self.get_by_id(id)
-        return self._build_sig_response(sig)
+    def get_sig_response_by_id(self, id: int) -> SIG:
```

Ruff (0.15.6)

[error] Function argument `id` is shadowing a Python builtin

(A002)